tests/decodecorpus.c

```diff
@@ -662,6 +674,12 @@ static U32 generateSequences(U32* seed, frame_t* frame, seqStore_t* seqStore,
                           MIN(frame->header.windowSize,
                               (size_t)((BYTE*)srcPtr - (BYTE*)frame->srcStart))) +
                          1;
+                if (info.useDict && (RAND(seed) & 1)) {
+                    /* need to occasionally generate offsets that go past the start */
+                    /* we still need to be within the windowSize however */
```

Comment:
I must have misunderstood that. So only `windowSize` bytes can be regenerated from the dictionary. Does that mean that we can also regenerate additional bytes from the current buffer? For example, can we have a match that overlaps with the dictionary for `windowSize` bytes and then overlaps with the current buffer for additional length? Are there any restrictions on that?